backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/dialect/model/impl/Elasticsearch8ModelDialect.java

```diff
@@ -6,25 +6,25 @@
  */
 package org.hibernate.search.backend.elasticsearch.dialect.model.impl;
 
-import org.hibernate.search.backend.elasticsearch.types.dsl.provider.impl.Elasticsearch8IndexFieldTypeFactoryProvider;
+import org.hibernate.search.backend.elasticsearch.types.dsl.provider.impl.Elasticsearch7IndexFieldTypeFactoryProvider;
 import org.hibernate.search.backend.elasticsearch.types.dsl.provider.impl.ElasticsearchIndexFieldTypeFactoryProvider;
-import org.hibernate.search.backend.elasticsearch.validation.impl.Elasticsearch8PropertyMappingValidatorProvider;
+import org.hibernate.search.backend.elasticsearch.validation.impl.Elasticsearch7PropertyMappingValidatorProvider;
 import org.hibernate.search.backend.elasticsearch.validation.impl.ElasticsearchPropertyMappingValidatorProvider;
 
 import com.google.gson.Gson;
 
 /**
- * The model dialect for Elasticsearch 8.x.
+ * The model dialect for Elasticsearch [8.0-8.11].
  */
 public class Elasticsearch8ModelDialect implements ElasticsearchModelDialect {
```

Comment:
we had a 8 dialect before all those changes for knn, so I've kept it as it was, even though we could just reuse the 7 one for Elasticsearch 8.0-8.11.